app/controllers/admin/users_controller.rb

```diff
@@ -1,7 +1,7 @@
 class Admin::UsersController < ApplicationController
   before_action :authenticate_user!
   before_action :check_admin
-
+  add_breadcrumb "admin", :admin_users_path
```

Comment:
avoid duplication of ``` add_breadcrumb "admin", :admin_users_path  ``` in every controller
check gem readme
they even show in this example
https://github.com/weppos/breadcrumbs_on_rails#restricting-breadcrumb-scope